features/product/managing_product_reviews/browsing_product_reviews.feature

```diff
@@ -0,0 +1,19 @@
+@managing_product_reviews
+Feature: Browsing product reviews
+    In order to know customer's opinion about my products
+    As an Administrator
+    I want to browse product reviews
+
+    Background:
+        Given the store has customer "Mike Ross" with email "ross@teammike.com"
```

Comment:
Do we need to create customers? IIRC we have a transformer that creates a customer if needed. We could than write 2 reviews for this product using different customers than the below comment can be ignored :)